### ISSUE TITLE
"1.1.4" version have some question want to ask

### ISSUE THREAD

**Issue by bau720123** [EddyVerbruggen/cordova-plugin-actionsheet#17]
I am using PGB 3.7 with 1.1.4 version plugin
these are my sample code
```
<script type="text/javascript">
function checkActionSheet()
{
 var options = 
 {
 'androidTheme': 1, //the default is 1，1：THEME_TRADITIONAL、2：THEME_HOLO_DARK、3：THEME_HOLO_LIGHT、4：THEME_DEVICE_DEFAULT_DARK、5：THEME_DEVICE_DEFAULT_LIGHT
 'title': 'how?',
 'buttonLabels': ['Facebook', 'Twitter'],
 'androidEnableCancelButton' : true,
 'winphoneEnableCancelButton' : true,
 'addCancelButtonWithLabel': 'cancel',
 'addDestructiveButtonWithLabel' : 'delete'
 };
window.plugins.actionsheet.show(options, callback);
setTimeout(function() { window.plugins.actionsheet.hide({}, onSuccess, onError); }, 3000);
};

function callback(buttonIndex) 
{
alert("buttonIndex：" + buttonIndex, null);
}

function onSuccess()
{
alert("success");
}

function onError()
{
alert("fail");
}
</script>
```

according to the latest document
http://plugins.cordova.io/#/package/nl.x-services.plugins.actionsheet

[question 1]
it said
```
'androidTheme': window.plugins.actionsheet.ANDROID_THEMES.THEME_HOLO_LIGHT, // default is THEME_TRADITIONAL
```
but in fact，it should be modify to this
```
 'androidTheme': 1, //the default is 1，1：THEME_TRADITIONAL、2：THEME_HOLO_DARK、3：THEME_HOLO_LIGHT、4：THEME_DEVICE_DEFAULT_DARK、5：THEME_DEVICE_DEFAULT_LIGHT
```

[question 2]
when I trigger "checkActionSheet" function
it will also trigger ```window.plugins.actionsheet.hide({}, onSuccess, onError);``` after three seconds
well...
but no mater which platform is
the "onSuccess" or "onError" will never trigger

[question 3]
after three seconds passed...
in Android and winphone，it will trigger "nothing"
but in iOS，it will automatically trigger "callback" javascript function and alert buttonIndex is "-1"
so...is the iOS's logical correct ?

**Comment by EddyVerbruggen** [comment]
Those theme constants should work as shown in the example. They refer to [these lines](https://github.com/EddyVerbruggen/cordova-plugin-actionsheet/blob/master/www/ActionSheet.js#L12-L18)

**Comment by bau720123** [comment]
hi @EddyVerbruggen 
thanks for your reply
but I really try many times in PGB
for example like that

[not work]
```
'androidTheme': window.plugins.actionsheet.ANDROID_THEMES.THEME_HOLO_LIGHT,
```
but in fact，it should be modify to this

[work]
```
 'androidTheme': 3, 
```

so...what's the detail with [question 2] and [question 3]

**Comment by bau720123** [comment]
hi @EddyVerbruggen 
problem solved in 1.1.7 version
thanks
